bazel-mode.el

```diff
@@ -0,0 +1,240 @@
+(defun sort-strings-longest-first (l)
+  (sort l #'(lambda (s1 s2) (> (length s1) (length s2)))))
+
+;; define several category of keywords
+(setq bazel-keywords
```

Comment:
Please use `defconst` for all constants. Though these lists should probably just be inlined, the `font-lock-keywords` constant is already obvious without specific names.

Reply:
I'll set them using defconst for now. I'm hesitant to undo the separation by mixing the various flavors of rules together until we make a decision about keeping the builtins or not.